src/sentry/incidents/logic.py

```diff
@@ -404,7 +389,7 @@ def calculate_incident_time_range(incident, start=None, end=None, windowed_stats
 
             start = end - timedelta(seconds=time_window * WINDOWED_STATS_DATA_POINTS)
 
-    retention = quotas.get_event_retention(organization=incident.organization) or 90
+    retention = quotas.get_event_retention(organization=incident.organization) or 90  # type: ignore[attr-defined]
```

Comment:
This `type:ignore` is needed because of the hack in `src/sentry/quotas/__init__.py`:

```python
backend.expose(locals())
```

Not sure there's a way around it without factoring out that hack and converting it to an actual method call.

Reply:
use `quotas.backend` directly

Reply:
Done: https://github.com/getsentry/sentry/pull/75541/commits/678a63f51f3837f98876cd837809e62bf6807c64